src/sass/_card.scss

```diff
@@ -10,11 +10,8 @@
 
   .card-image {
     background-repeat: no-repeat;
-    -webkit-background-size: cover;
-    -moz-background-size: cover;
-    -o-background-size: cover;
     background-size: cover;
-    background-position: center center;
+    background-position: 50%;
```

Comment:
We can make this `center center` again. https://cssreference.io/property/background-position/